inference-engine/samples/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-cmake_minimum_required (VERSION 2.8.12)
+cmake_minimum_required (VERSION 3.13)
```

Comment:
no. Samples still require only 2.8.12

Reply:
Yes. That was something decided on various meetings that samples stop support older cmake versions as this introduces unnecessary complexity to their support.

Reply:
anyway, currently we don't anything that require 3.13.